src/components/ErrorMessage/ErrorMessage.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import { ApiError } from '../../types/ApiError';
+
+interface Props {
+  removeErrorMessage: () => void;
+  errorType: ApiError;
+}
+
+export const ErrorMessage: React.FC<Props> = ({
+  removeErrorMessage,
+  errorType,
+}) => {
+  let message = '';
+
+  switch (errorType) {
+    case ApiError.Get:
+      message = 'Unable to load todos';
+      break;
+    case ApiError.Post:
+      message = 'Unable to add todo';
+      break;
+    case ApiError.Patch:
+      message = 'Unable to update a todo';
+      break;
+    case ApiError.Delete:
+      message = 'Unable to delete a todo';
+      break;
+    default:
+      message = 'Something goes wrong';
+  }
```

Comment:
it's bad practice
let's prettify your code

1. Don't create unnecessary variables with `let`
2. Don't put blocks like `for / while / switch` on the top of the body in your component
3. You can return some message at once in function

example 

```
const errorMessage = useMemo(() => {
  switch (errorType) {
    case ApiError.Get:
      return 'Unable to load todos';

    case ApiError.Post:
      return 'Unable to add todo';
  }
}, [errorType])

if (errorType === ApiError.None) {
  return null;
}
```